pkg/hostagent/status.go

```diff
@@ -68,6 +68,7 @@ func (agent *HostAgent) RunStatus() {
 	http.HandleFunc("/status", func(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Content-Type", "application/json")
 		agent.indexMutex.Lock()
+		agent.ipamMutex.Lock()
```

Comment:
What is the relevance of adding a second mutex here? You already have one mutex which you are locking and unlocking at the same place...